src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -22,12 +27,16 @@ public class ViewLoadedParams : ReadyParams
         private readonly DynamoViewModel dynamoViewModel;
         public readonly Menu dynamoMenu;
 
+        public IWatch3DView BackgroundPreView { get { return dynamoView.BackgroundPreview; } }
```

Comment:
See my comment below regarding moving this interface to the view model instead. That way we get access to the `Watch3DViewModel` that we need through the `DynamoViewModel` which we already have :)